src/pray.c

```diff
@@ -1554,8 +1554,11 @@ dosacrifice()
             } else { /* super big win */
                 adjalign(10);
                 u.uachieve.ascended = 1;
-                pline(
-               "An invisible choir sings, and you are bathed in radiance...");
+                /* you might be able to see the invisible choir! */
+                pline("%s sings, and you are bathed in radiance...",
+                      Hallucination ? "The fat lady"
+                          : (See_invisible && !Blind ? "A visible choir"
```

Comment:
Will it make more sense for it to just say "A choir" instead of having the extra "visible" in there?

Reply:
I figured it would be less noticeable this way, and a subtle YAFM is a good YAFM in my book.

Reply:
Perhaps "transparent" rather than "visible"?